src/go/rpk/pkg/cli/cmd/redpanda/start.go

```diff
@@ -171,6 +173,25 @@ func NewStartCommand(
 				conf.Redpanda.KafkaApi = kafkaApi
 			}
 
+			proxyAddr = stringSliceOr(
+				proxyAddr,
+				strings.Split(
+					os.Getenv("REDPANDA_PANDAPROXY_ADDRESS"),
+					",",
+				),
+			)
+			proxyApi, err := parseNamedAddresses(
+				proxyAddr,
+				config.DefaultProxyPort,
+			)
+			if err != nil {
+				sendEnv(fs, mgr, env, conf, err)
+				return err
+			}
+			if proxyApi != nil && len(proxyApi) > 0 {
+				conf.Pandaproxy.PandaproxyAPI = proxyApi
```

Comment:
`conf.Pandaproxy` might be nil. In that case, should it be init'd with the default values and then assigned the addresses?

Reply:
Make sense.